cypress/integration/calculator.spec.js

```diff
@@ -0,0 +1,102 @@
+describe('Calculator test', () => {
+  
+  const range = (...args) => {
+    const { length } = args
+    
+    if (length === 0 || length > 2) throw new Error('인자값은 반드시 한개 또는 두개 입니다.')
+    
+  
+    if (length === 1 && typeof args[0] === 'number') {
+      const stack = []
+      for (let i = 0; i < args[0]; i++) {
+        stack.push(i)
+      }      
+      return stack
+    }
+  
+    if (length === 2 && (typeof args[0] === 'number' && typeof args[1] === 'number')) {
+      
+      const stack = []
+      const [start, end] = [args[0], args[1]]
+      for (let i = start; i < end; i++) {
+        stack.push(i)
+      }
+      return stack
+    }
+  }
+
+
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('초기 계산기 값은 0이다.', () => {
+    cy.get('#total').should('contain.text', '0')
+  })
+
+  it('2개의 숫자에 대해 덧셈이 가능하다', () => {
+    const [first, second] = [range(10)[Math.floor(Math.random() * 10)], range(10)[Math.floor(Math.random() * 10)]]
+    
+    cy.get('.digit').contains(`${first}`).click()
```

Comment:
contains안에 바로 first를 넣어도 되지 않을까요?
```suggestion
    cy.get('.digit').contains(first).click()
```

Reply:
타입이 문자열일 때만 가능한지 알았습니다. 수정하겠습니다!